lib/checkPullRequestReview.js

```diff
@@ -381,9 +382,63 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {string[]} assignees
+ * @param {string} comment
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+
+//This is a function to check if there are more than 24 Hours with the Pr
+const isMaxReviewPeriodExceeded = async (review) => {
+  let timeOfReviewSubmission = review.submitted_at;
+  timeOfReviewSubmission = new Date(timeOfReviewSubmission);
+  let currentTime = new Date.now();
+  let diff = Math.abs(timeOfReviewSubmission - currentTime);
+  return diff > MAX_REVIEW_PERIOD;
```

Comment:
`diff` would be in milliseconds right?
`MAX_REVIEW_PERIOD` is in hours so `diff` will always be greater than `MAX_REVIEW_PERIOD`.

Reply:
This one I missed Sir, I will convert it in milliseconds. Thanks for pointing it out.